_episodes/11-hpc-intro.md

```diff
@@ -40,83 +34,61 @@ they started:
-
-* We will learn to use the UNIX command line (also known as Bash).
-* We will use our new Bash skills to connect to and operate a high-performance computing
-  supercomputer.
-
-The skills we learn here have other uses beyond just HPC - Bash and UNIX skills are used everywhere,
-be it for web development, running software, or operating servers. It's become so essential that
-Microsoft now ships as [part of Windows][_ms-ubuntu])! Knowing how to use Bash and HPC systems will
-allow you to operate virtually any modern device. With all of this in mind, let's connect to a
-cluster and get started!
-
-[_ms-ubuntu]: https://www.microsoft.com/en-us/store/p/ubuntu/9nblggh4msv6
+
+> # And what do you do?
+> 
+> Talk to your neighbour, office mate or [rubber duck](https://rubberduckdebugging.com/) about your research. How does computing help you do your research? 
+> How could more computing help you do more or better research?
+{: .callout }
+
+
+## Doing Analysis or Running Code
+
+### A standard Laptop for standard tasks
+
+Today, people coding or analysing data typically work with laptops.
+ 
+{% include figure.html url="" max-width="20%" file="/fig/200px-laptop-openclipartorg-aoguerrero.svg" alt="A standard laptop"
+caption="" %}
+
+While the following discussion may appear silly at first glance, let's dissect what resources programs running on a laptop require:
+- the keyboard and touchpad is used to provide flags and options
+- the display is shows progress and results
+- the laptops internal computing power and memory is used to perform calculations
+- offen GUI style human interfaces are employed
+
+### When tasks take too long
+
+When the task to solve become more heavy on computations, the operations are typically out-sourced from the local hardware in the laptop. Take for example the task to find the directions for your next business trip. The capabilities of your laptop are typically not enough to calculate that route spontaneously. So you use website, which in turn runs on a server that is almost exclusively no in the same room than you are.
+
+{% include figure.html url="" max-width="20%" file="/fig/servers-openclipartorg-ericlemerdy.svg" alt="A rack half full with servers"
+caption="" %}
+
+Note here, that a server is mostly a noisy computer mounted into a rack cabinet which in turn resides in a data center. The internet made it possible that these data centers do not require to be nearby your laptop. 
+What people call **the cloud** is mostly a web-service where you can rent such servers by providing your credit card details and by clicking together the specs of this remote resource.
+
+The server itself has no direct display or input methods attached to it. But most importantly, it has much more storage, memory and compute capacity than your laptop will ever have. In any case, you need a local device (laptop, workstation, mobile phone or tablet) to interact with this remote machine, people typically call 'a server'. 
+
```

Comment:
I like this progression from: your computer -> computers that you rent that are elsewhere -> groups of computers